src/include/nfs/ui.rb

```diff
@@ -475,6 +475,8 @@ def GetFstabEntry(fstab_ent, existing)
       UI.CloseDialog
       Wizard.RestoreScreenShotName
 
+      return nil if ret == :cancel
```

Comment:
looks like only possible ret here are cancel and ok, so code that follows can be simplified ( removing condition and also removing remaining nil )

Reply:
That entire code in that entire module can be vastly simplified. It's huge functions, poorly readable code, poorly testable, and YCP zombies are all over the place.

But this is a hot fix for a very embarrassing crash; the objective is to minimize the diff to avoid introducing new problems.